stdlib/gc.mli

```diff
@@ -149,7 +149,30 @@ type control =
         out variations in its workload. This is an integer between
         1 and 50.
         Default: 1. @since 4.03.0 *)
-}
+
+    custom_major_ratio : int;
+    (** A percentage relative to the major heap size. This parameter
+        controls the major GC speed as a function of memory allocated
```

Comment:
Why not reuse the text from the manual ("Target ratio of floating garbage to
major heap size for out-of-heap memory held by custom values
located in the major heap...")?

Reply:
Done.